geopandas/base.py

```diff
@@ -409,6 +447,10 @@ def boundary(self):
         2                             GEOMETRYCOLLECTION EMPTY
         dtype: geometry
 
+        See also
+        --------
+        GeoSeries.exterior : outer boundary
```

Comment:
What's actually the diffference? 
I think that 1) `exterior` returns a LinearRing, while `boundary` returns a LineString (in case of a polygon) and 2) `exterior` only works for a `Polygon`.

(should maybe mention at least the second?)

Reply:
`exterior` does not give you interior rings while `boundary` does. That is the main practical difference I used so far. The fact that `exterior` works on polygons only is included in the documentation of `exterior`.